support-frontend/assets/components/personalDetails/personalDetails.tsx

```diff
@@ -35,6 +50,8 @@ export type PersonalDetailsProps = {
 	onLastNameChange: (lastName: string) => void;
 	signOutLink: React.ReactNode;
 	contributionState: React.ReactNode;
+	overrideHeadingCopy?: string;
+	hideDetailsHeading?: true;
```

Comment:
Does this always default to hiding even if false supplied? Don't suppose this situation is likely?